create.go

```diff
@@ -284,3 +292,58 @@ func createPIDFile(pidFilePath string, pid int) error {
 
 	return nil
 }
+
+// copyIfNeeded copies the cpuset.cpus and cpuset.mems from the parent
+// directory to the current directory if the file's contents are 0
+func copyIfNeeded(current, parent string) error {
```

Comment:
I would just call that one `copyParentCpuSet()`
The `IfNeeded` part is not necessary.